### DISCUSSION TITLE
Roman numeral ligatures with final j

### DISCUSSION THREAD

**Discussion by TheKnightWho** [psb1558/Junicode-font#118] [Ideas]
Hi - could I please request hlig variants for the lowercase Roman numeral ligatures that end with a final j? My preference would be for these to be alternatives to the current ligatures rather than a whole new set involving the use of the character 'j', for the sake of text search etc.

**Comment by psb1558** [comment]
How useful would these be, in fact? There are only a few roman number ligatures. If you're thinking of using them in editions, they won't get you very far.

It would be  preferable, I think, to have a variant of i that would show as j in final position.

**Comment by psb1558** [comment]
Here's what I have in mind (and I _think_ I've managed to make it work). Apply cv18[4] to a roman number and, if it ends with i, the i will show as j. But it's actually a glyph `i.jform` which will be searchable as i. I was thinking that it should be applied only to roman numbers (not to words like _ingenii_), but it occurred to me that in MSS when a word ends in _-ii_, the final i often has the form of a j. So I may relabel this feature to something like "j-shaped i in final position."

"<img width="716" alt="image" src="https://user-images.githubusercontent.com/16215767/157766124-4cec5a27-3ea5-41fb-be86-1325a62b3814.png">

**Comment by psb1558** [comment]
Surveying roman numbers in various MSS, I've settled on a couple of features, available in the next version (1.045). cv18[4] ("ii to ij at word-end") will change **i** to **j** (or, more exactly, a j-shaped **i**) at the end of a word (or roman number), but only when preceded by **i**. cv18[5] ("i to j at word-end") will change **i** to **j** at the end of a word no matter what.

A **j** produced by this feature will always be searchable as **i**.

**Reply by TheKnightWho** [reply]
That's really great - thank you!
